### PR TITLE
New status icon for support of features by OTel itself

### PR DESCRIPTION
**Pull request by rkbodenner**
Some features aren't available in New Relic + OTel yet because the OTel spec or implementations don't quite support what's needed. We need a way to differentiate those from features that we at New Relic have not yet implemented.

We will need some review from a few stakeholders before merging: @rhetoric101 @reese-lee to name a few

### CODE REVIEW

**Comment by rhetoric101** [comment]
Docs hero, could you take an initial pass through these changes and pass to me?

**Comment by x8a** [comment]
Hey, @rhetoric101! Can you make sure this one is good to go?

**Comment by rhetoric101** [comment]
The build failed after I merged develop into this branch, so I am temporarily closing this PR to kick off a new build.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 1]
```diff
@@ -366,27 +367,25 @@ Distributed tracing is supported today by our agents and OpenTelemetry. Both pro
...
+  New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 Limited support
 
-  In order for data from an OpenTelemetry service to populate the **Transactions** page, the spans must be `SpanKind` of `server` or `consumer`. These spans represent requests processed by the service, which may be part of a larger trace involving more services. You may have to set this attribute manually if you want this data in the Transactions UI (see [Transactions page](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-transactions-page)).
+  Transactions are a New Relic concept, representing an aggregation of requests to a particular endpoint. For OpenTelemetry we map this concept using trace data (see [Transactions](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic#transactions)). Metric data can also power the Transactions experience, but is only produced by some OTel language agents. All New Relic agents produce transaction metrics, including additional metrics on aggregate performance of transaction segments.
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:372]: Added more on metrics for Transactions here

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 2]
```diff
@@ -449,56 +448,60 @@ The features of the database page will vary due to the following:
...
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
+
+New Relic agents offer automatic log collection by default for services. OpenTelemetry can be configured to send logs from a variety of sources.
 
 ## Service map [#service-maps]
 
 New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
 
-This map displays an entire set of interconnected services for a given service. The interconnected services must be actively reporting to appear in this map. Compare with these features: related entities and automap.
+The Service Map, accessible when looking at a single service in the UI, displays an entire set of entities connected to that service and is currently only supported by New Relic agents. The interconnected services must be actively reporting to appear in this map.
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:483]: A little more clarity on what "service map" specifically means. More consolidation of the maps content in a future PR.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 3]
```diff
@@ -519,18 +522,20 @@ OpenTelemetry metrics release candidate was announced earlier in May, with gener
...
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
 
-    The New Relic agent reports metrics (as well as spans and events) scoped to an individual transaction or endpoint.
+    The Transactions page works with New Relic agent metrics, or with metrics for HTTP services reported by OpenTelemetry. OTel instrumentation, however, does not consistently or by default report transction/endpoint metrics.
+    
+    All New Relic language agents report metrics (as well as spans and events) scoped to an individual transaction or endpoint.
 
-    Default OpenTelemetry SDK instrumentation does not consistently report such metrics. The semantic conventions for endpoint-level data, for example `http.target`, are still considered experimental.
+    Default OpenTelemetry SDK instrumentation does not consistently report such metrics. The semantic conventions for endpoint-level data, for example `http.route`, are still considered experimental.
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:531]: This was an important recent change to align with the OTel spec, which Jack Berg moved forward

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 4]
```diff
@@ -694,133 +698,136 @@ New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
...
 
 New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 🟡 Limited support
 
-This view shows:
+This view shows, for APM and/or OpenTelemetry services or hosts:
 
-* Relationships between services (APM and OpenTelemetry services)
-* Relationships between services and infrastructure (not supported for OpenTelemetry services)
+* Relationships between services
+* Relationships between services and infrastructure
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:825]: This and the next line are now up-to-date with respect to how entity relationships are created. OTel hosts work, as do uninstrumented external services. It's just service-to-database that's not done yet for OTel.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 5]
```diff
@@ -1061,9 +1071,9 @@ Conditions, violations, incidents, and issues all work for any entity.
     id="cust-evnts"
     title="Custom events"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic 🟡
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
 
-    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).
+    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the experimental OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:1076]: @rhetoric101 This needs tweaking with Mark's help before we publish

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:531]: @rkbodenner Looks good to me!

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 4 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:825]: @rkbodenner Looks good to me!

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 6]
```diff
@@ -69,70 +69,71 @@ We recommend you start with the table at the beginning that shows high-level com
     </tr>
     <tr>
       <td>
-        Measure and track SLIs/SLOs
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:71]: Relocated this later in the list to keep APM things grouped together up top

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 7]
```diff
@@ -69,70 +69,71 @@ We recommend you start with the table at the beginning that shows high-level com
...
       </td>
       <td>
-        ✅
+        👀 Better out-of-the-box experience with New Relic or Pixie instrumentation
       </td>
     </tr>
     <tr>
       <td>
-        Monitor Kubernetes
+        Monitor hosts
```
Comment (rkbodenner) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:127]: Added hosts, per Mark's feedback

**Comment by rkbodenner** [comment]
With this latest commit, I think we're done with content changes. Can reorder the detailed comparisons into groups and link to the high-level descriptions in a future PR.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 5 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx]: @rkbodenner I chopped Mark's information in half, and put the second half here.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 6 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:71]: Sounds good!

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 7 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:127]: Looks good!

**Review by rhetoric101 — APPROVED** [review]
@rkbodenner thanks for making these improvements to this comparison!

**Pull request merged**